.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ commands:
       steps:
         - run:
             name: Update memory setting
-            command: sed -i "s/org.gradle.jvmargs=.*/org.gradle.jvmargs=-Xmx1537m -XX:+HeapDumpOnOutOfMemoryError/" gradle.properties
+            command: sed -i "s/org.gradle.jvmargs=.*/org.gradle.jvmargs=-Xmx3500m -XX:+HeapDumpOnOutOfMemoryError/" gradle.properties
```

Comment:
Right! It didn't solve the issue, but I thought that there's no downside on keeping it since we have 4 GB containers and all their resources can be spent on the current task.

Reply:
It turned out it has downsides: it didn't left enough memory to JDK to run. I've reverted it.